core/src/com/group66/game/cannon/Ball.java

```diff
@@ -1,76 +1,159 @@
 package com.group66.game.cannon;
 
-import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import com.badlogic.gdx.graphics.g2d.TextureRegion;
 import com.badlogic.gdx.math.Circle;
 import com.group66.game.helpers.AssetLoader;
 
+/**
+ * A basic Ball class.
+ */
 public class Ball {
 	
+	/** The Constant that represents a BLUE ball. */
 	public static final int BLUE = 0;
+	
+	/** The Constant that represents a GREEN ball. */
 	public static final int GREEN = 1;
+	
+	/** The Constant that represents a RED ball. */
 	public static final int RED = 2;
+	
+	/** The Constant that represents a YELLOW ball. */
 	public static final int YELLOW = 3;
```

Comment:
No, because these are specific to the Ball class, the Config class is for game configurations. The Ball, BallManager, Cannon class etc should be "units" of their own (stand alone) and you should be able to import those into another project without changing the code.